docs/pages/docs/handbook/linting/basics.mdx

```diff
@@ -0,0 +1,52 @@
+# Linting in a monorepo
+
+Linting in a monorepo can be tricky. Most of your workspaces will likely contain code that needs to be linted - so working out the most efficient way to lint them is tough.
+
+In this guide, we'll propose a method that plays to Turborepo's strengths:
+
+- Running lint tasks _inside_ the workspaces, not from root
+- Sharing as much config as possible between workspaces
+
+## Running tasks
+
+We recommend specifying a single `lint` task inside your `turbo.json`.
+
+```json filename="turbo.json"
```

Comment:
Perhaps such an annotation could also be used in `docs/pages/docs/handbook/workspaces.mdx` on lines `31`, `48`, and `59` (the code blocks inside the Tabs). The filename for the first two should be `package.json` and `pnpm-workspace.yaml` for the second.